moonbeam-types-bundle/index.ts

```diff
@@ -348,7 +348,7 @@ export const moonbeamDefinitions = {
         AccountId: "EthereumAccountId",
         AccountInfo: "AccountInfoWithTripleRefCount",
         Address: "AccountId",
-        AuthorId: "AccountId",
+        AuthorId: "RelayChainAccountId",
```

Comment:
Why is the AuthorId a RelayChainAccountId ?

Reply:
The author id is a relaychain address, right @JoshOrndorff ?

Reply:
I'm not surprised that this "works" because a relay chain account and a NimbusId are both 32-bytes of raw data, but I don't think this type is semantically correct.

This should be the same type that is used for other session keys. Looking at Apps, it seems @Jacogr is indeed using the `AccountId` type for session keys (for example [here](https://github.com/polkadot-js/api/blob/ed5f2673cebc30094c219f4d7b0ceb90d981392f/packages/types/src/interfaces/session/types.ts#L32)) but I believe that is not really correct because soe chains can have a different AccountId type. For example Moonbeam.

Maybe a good middle ground for this PR would be `AuthorId: "AccountId32"`. That has the weakness of still being tied to a not-really-correct accountId type, but at least it is clear that this isn't the relay chain account id or the moonbeam account id. It is a different 32 byte thing.

Reply:
Thanks :)

Reply:
AccountId32 is not defined in the api though. What about plain old "H256" ?

Reply:
or "AccountId32":"H256"